doc-index-updater/overlays/local/keys.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Redis credentials...
+REDIS_KEY=$(az redis list-keys \
+    --resource-group mhra-products-development \
+    --name doc-index-updater-non-prod \
```

Comment:
Should this one be `doc-index-updater-dev`?

(And maybe similar for the service bus namespaces below?)

Reply:
yep, will fix now. Thanks